client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -249,9 +220,22 @@ class ActivityLogItem extends Component {
 	}
 
 	renderHeader() {
+		const { actor = {} } = this.props.log;
+		const {
```

Comment:
Maybe it would make sense to extract out actor's details within the actor component itself, rather than doing it outside? I'm fine either way.